src/Keyboard.js

```diff
@@ -0,0 +1,367 @@
+/* eslint-disable import/extensions */
+import Button from './Button.js';
+import buttonConfig from './buttonConfig.js';
+import { caseConfig, specialBtn } from './constants.js';
+
+export default class Keyboard {
+  constructor() {
+    this.keyboard = null;
+    this.textarea = null;
+    this.container = null;
+    this.lang = localStorage.getItem('lang') || 'En';
+    this.case = caseConfig.lower;
+    this.pressedCtrlAlt = new Set();
+  }
+
+  renderKeyboard() {
+    this.container = document.createElement('div');
+    this.container.id = 'container';
+    this.container.classList.add(this.lang);
+
+    const title = document.createElement('h1');
+    title.classList.add('title');
+    title.textContent = 'Виртуальная клавиатура';
+
+    this.textarea = document.createElement('textarea');
+    this.textarea.classList.add('textarea');
+
+    this.keyboard = document.createElement('div');
+    this.keyboard.id = 'keyboard';
+    this.keyboard.classList.add(this.case);
+
+    this.addButtonsToKeyboard();
+
+    const helpText = document.createElement('p');
+    helpText.classList.add('help-text');
+
+    const helpSpanEn = document.createElement('span');
+    helpSpanEn.classList.add('En');
+    helpSpanEn.textContent = 'The keyboard is created in Windows. To switch the language, use Ctrl + Alt. Have a nice day! ;)';
+
+    const helpSpanRu = document.createElement('span');
+    helpSpanRu.classList.add('Ru');
+    helpSpanRu.textContent = 'Клавиатура создана в ОС Windows. Для переключения языка используйте Ctrl + Alt. Хорошего дня! ;)';
+
+    helpText.append(helpSpanEn, helpSpanRu);
+
+    this.container.append(title, this.textarea, this.keyboard, helpText);
+
+    document.body.prepend(this.container);
+  }
+
+  addButtonsToKeyboard() {
+    for (let i = 0; i < buttonConfig.lowerEn.length; i += 1) {
+      const row = document.createElement('div');
+      row.classList.add('row');
+
+      for (let j = 0; j < buttonConfig.lowerEn[i].length; j += 1) {
+        const button = new Button(i, j).createButton();
```

Comment:
1. it's better to separate Button creation and calling a method: 
const button = new Button(i, j);
row.append(button.createButton());

2. Try to avoid variables like `i`, `j`. It should be something meaningful: `key`, `name`

Reply:
Done)